front/app/api/campaigns/util.ts

```diff
@@ -1,5 +1,17 @@
-import { ICampaignData } from './types';
+import { ICampaignData, CampaignContext } from './types';
 
 export function isDraft(campaign: ICampaignData) {
   return campaign.attributes.deliveries_count === 0;
 }
+
+export function getCampaignsContextPath({
+  projectId,
+  phaseId,
+}: CampaignContext) {
+  if (projectId) {
+    return `projects/${projectId}/campaigns`;
+  } else if (phaseId) {
+    return `phases/${phaseId}/campaigns`;
```

Comment:
Is there every a chance that both a Project ID and Phase ID are passed? If so, we might want to have the Phase ID check first.